src/Core/src/Handlers/DatePicker/DatePickerHandler.iOS.cs

```diff
@@ -16,7 +16,7 @@ protected override MauiDatePicker CreatePlatformView()
 
 			_picker = new UIDatePicker { Mode = UIDatePickerMode.Date, TimeZone = new NSTimeZone("UTC") };
 
-			if (PlatformVersion.IsAtLeast(14))
+			if (OperatingSystem.IsIOSVersionAtLeast(14)) // Seems supported version is 13.4
```

Comment:
In this case, should the check be `IsIOSVersionAtLeast(13.4)`

Reply:
It could be, that is the question 😄